storage.go

```diff
@@ -216,3 +245,20 @@ func (s *Storage) write(ctx context.Context, path string, r io.Reader, size int6
 	}
 	return size, nil
 }
+
+func (s *Storage) writeAppend(ctx context.Context, o *Object, r io.Reader, size int64, opt pairStorageWriteAppend) (n int64, err error) {
+	rp := o.GetID()
+	nextPos := o.MustGetAppendOffset()
```

Comment:
`MustGetAppendOffset` will panic if append offset does not exist.

In production code, we usually `offset, ok := o.GetAppendOffset()` and check the returning value to decide whether to return an error or not.

Reply:
Which error is suitable when the append offset is unset? Do I need to definite error code?

Reply:
For now, let's just use `fmt.Errorf("append offset is not set")`. We will add this error in `go-storage`.